### PR TITLE
added list README

### PR DESCRIPTION
**Pull request by Bhupesh-V**
**Fixes issue:**

- Added README for Linked List Data Structure.

**Changes:**

### CODE REVIEW

File: code/data_structures/src/linked_list/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Linked Lists
+Description
+---
```
Comment (arnavb) [inline code/data_structures/src/linked_list/README.md]: While this syntax for headers work, could you use the `##` syntax, which is more common as well?

Comment (Bhupesh-V) [reply code/data_structures/src/linked_list/README.md]: added ```##``` for h2 level heading

File: code/data_structures/src/linked_list/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Linked Lists
+
+## Description
+---
+A linked list is a linear data structure, in which the elements are not stored at contiguous memory locations. The elements in a linked list are linked using pointers.
+In simple words, a linked list consists of nodes where each node contains a data field and a reference(link) or pointer to the next node in the list.
+
+## Functions
+---
```
Comment (arnavb) [inline code/data_structures/src/linked_list/README.md]: Could you remove this HRs?

**Comment by arnavb** [comment]
djldk 2

**Pull request merged**